### PR TITLE
Todolist project

### PR DESCRIPTION
**Pull request by SumitKanth**
> **Note** to the contributors:
>
> Have you read the Contribution Guidelines? If not, check [CONTRIBUTING.md](/CONTRIBUTING.md) file first.

## What kind of change does this PR introduce? (select / add custom
new project

## Description 
Added todolist project, made on react. In this user can write there Todos, if they want to update their Todo they can do that also and also delete the todo. 

- [x] I've read the contribution guidelines.
- [ ] I've mentioned my project in the Readme as per guidelines.
- [x] I've checked that my project is not a duplicate project and I have cross checked the exisiting projects in this repository.

## Related Issues
Fixes Issue #7

### CODE REVIEW

**Comment by Greeshma2903** [comment]
Awesome! Can you mention your project details in the Readme? (underneath Project List section)

**Comment by SumitKanth** [comment]
> Awesome! Can you mention your project details in the Readme? (underneath Project List section)

where ?

**Comment by Greeshma2903** [comment]
[As mentioned here](https://github.com/Design-and-Code/Quick-projects/blob/95c2134d11566d78b50f24cd7d29cee6f1fec6fc/CONTRIBUTING.md?plain=1#L94), you can add it in the table under `reactJS` under [Projects List](https://github.com/Design-and-Code/Quick-projects#projects-list-) section in README.

**Comment by SumitKanth** [comment]
@Greeshma2903 is it okii ? 👀 

Screenshot (571)

**Comment by Greeshma2903** [comment]
> @Greeshma2903 is it okii ? 👀
> 
> Screenshot (571)

Yes

**Comment by SumitKanth** [comment]
@Greeshma2903 can you review it

**Review by Greeshma2903 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -56,8 +56,20 @@ NOTE:
...
+
+### [ReactJS](/react/)
+
+| Project Name (as is) | Description                                             |
+| -------------------- | ------------------------------------------------------- |
+| react-shopping-cart  | Webpage to store social links, using TailwindCSS and JS |
+| Todo-List  | Webpage to write your todo's, using react |
+
+<!-- ### [go](/go/) -->
+=======
```
Comment (Rajkumar-justcoder) [inline README.md:69]: also here ig

**Comment by SumitKanth** [comment]
@Greeshma2903  @Rajkumar-justcoder now ?

Screenshot (574)

**Comment by Greeshma2903** [comment]
> @Greeshma2903 @Rajkumar-justcoder now ?
> 
> Screenshot (574)
Yes.
Also, please add the project description to only one ReactJS table (the one below where other projects are mentioned). 

https://github.com/Design-and-Code/Quick-projects#reactjs

**Comment by SumitKanth** [comment]
@Greeshma2903 now ?

Screenshot (581)

**Comment by Greeshma2903** [comment]
> @Greeshma2903 now ?
> 
> Screenshot (581)

Umm, I can't see any changes 😅

**Comment by SumitKanth** [comment]
@Greeshma2903 what to do 👀

**Comment by Greeshma2903** [comment]
I hope the changes are clear now 😅 I'll go ahead and _close_ this PR.

**Pull request closed without merging**